packages/tocco-ui/src/EditableValue/typeEditors/DecimalEdit.spec.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react'
+import {mount} from 'enzyme'
+import {IntlStub} from 'tocco-test-util'
+
+import DecimalEdit from './DecimalEdit'
+
+describe('tocco-ui', () => {
+  describe('EditableValue', () => {
+    describe('typeEditors', () => {
+      describe('DecimalEdit ', () => {
+        it('should render DecimalEdit', () => {
+          const number = 1234567.89
+          const wrapper = mount(
+            <DecimalEdit value={number} options={{intl: IntlStub}} />
+          )
+          expect(wrapper.find('NumberFormat')).to.have.length(1)
```

Comment:
expect(wrapper.find('input')).to.have.length(1) would be a better verification of that i guess.

also number in a own const is a bit superfluous if you dont really test anything with it. 

-> <DecimalEdit value={1} options={{intl: IntlStub}} />